assets/javascripts/discourse/connectors/topic-navigation/no-answer.js.es6

```diff
@@ -0,0 +1,29 @@
+import { later } from "@ember/runloop";
+
+// 7 days in milliseconds
+const MAX_DURATION_WITH_NO_ANSWER = 7 * 24 * 60 * 60 * 1000;
+
+export default {
+  setupComponent(args, component) {
+    this.set("show", false);
+
+    later(() => {
```

Comment:
what happens if I move to another route (admin for example) during this 2 seconds window?

Reply:
Nothing happens, no warning or error.

Reply:
Surprising but ok, if nothing you can see happens, it's probably good enough for now. I don't really like this kind of long later handlers, but fine.

Thanks for checking.

Reply:
Does the handler leak in that case?

Reply:
Since it was not a big deal, I just added a check for that to return early.